### ISSUE TITLE
Question: Best way to pass variables like %UNITY_WEBGL_BUILD_URL% to the React app?

### ISSUE THREAD

**Issue by Vixxd** [jeffreylanters/react-unity-webgl#107]
I am a Unity developer but this is my first time using React, so apologies if this is an obvious question! This plug in is great - I've built my React app to a WebGL template within a Unity project, and I'm able to locally test all I need.

However, I'd like other people to be able to build this Unity project using the React WebGL template, without having to ensure the name and location matches `Build/Build.json` each time. 

What's the best way (if any) to pass the the final populated values of things like `%UNITY_WEBGL_BUILD_URL%`, `%UNITY_WIDTH %`, etc. in `index.html` to the React app? (https://docs.unity3d.com/Manual/webgl-templates.html) 
 
I did leave this in my `index.html`:

```
<script> var gameInstance = UnityLoader.instantiate("gameContainer", "%UNITY_WEBGL_BUILD_URL%"); </script>
```

And remove the container setup from App.js in the [sample project](https://github.com/jeffreylanters/react-unity-webgl-test/blob/57a8db3d28cdce8da897df0a5457da0db9c6a30d/src/App.js#L66):

```
<Unity unityContent={this.unityContent} /> 
```
And it did work, but felt a bit hacky - I don't know enough about React to understand how the index.html-instantiated `UnityLoader` was responding just as I expected the App.js `this.unityContent` to.  I don't think the variables need to explicitly be within that <script...> tag, just as long as it's somewhere within index.html means Unity should replace the appropriate string at build time iirc.

TL;DR I'd like to pass the variables Unity sets in index.html (e.g. %UNITY_WEBGL_BUILD_URL%) to instantiate unityContent in the React app, rather than using UnityLoader.instantiate(...) in index.html.
